index.html

```diff
@@ -43,15 +43,17 @@
     <script>
       var thisMonday = moment().day(1);
       var frame = document.querySelector('.this.week iframe');
-      frame.src = 'https://etherpad.mozilla.org/PDXLunchWeekOf' + thisMonday.format('DDMMMM');
+      var thisEtherpad = 'https://etherpad.mozilla.org/PDXLunchWeekOf' + thisMonday.format('DDMMMM');
+      frame.src = thisEtherpad;
       var thisWeekDate = document.querySelector('.this-week-date');
-      thisWeekDate.textContent = thisMonday.format('MMMM DD');
+      thisWeekDate.textContent = "<a href="+thisEtherpad+">"thisMonday.format('MMMM DD')+"</a>";
```

Comment:
Missing a "+" before `thisMonday`.

Also, assigning HTML to `textContent` won't work, you're looking for `innerHTML` instead. But I'd just put the link in the HTML anyway and use `querySelector` to find it.